.pipelines/e2e-with-billing.yml

```
@@ -17,7 +17,7 @@ stages:
         subscription: $(e2e-subscription)
         azureDevOpsE2EJSONSPN: $(aro-v4-e2e-devops-spn)
         aroVersionStorageAccount: $(aro-version-storage-account)
-        rpMode: int
+        rpMode: ''
```

INT pipeline directly runs this template.  There is a variable set though for this.

```suggestion
        rpMode: $(RP_MODE)
```